src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/InitializeHandler.cs

```diff
@@ -92,7 +92,7 @@ internal class InitializeHandler : IRequestHandler<InitializeParams, InitializeR
                 HoverProvider = true,
                 DefinitionProvider = true,
                 DocumentHighlightProvider = true,
-                RenameProvider = true,
+                RenameProvider = new RenameOptions() { PrepareProvider = true },
```

Comment:
The HtmlCSharp server is only used if the `SingleServerSupport` feature flag is off, which is not the default. I suspect this isn't the change you intended to make.